test/test.js

```diff
@@ -96,9 +96,8 @@ describe('auto-check element', function() {
       const input = document.querySelector('input')
       input.value = 'hub'
       input.dispatchEvent(new InputEvent('change'))
-      input.addEventListener('auto-check-loadstart', () => {
```

Comment:
I'm gonna look over more of the tests and see if we have more broken ones and make sure they actually fail when they should. This should have been using the `done` callback to make sure that it fails correctly.